databricks/koalas/series.py

```diff
@@ -491,9 +491,38 @@ def dropna(self, axis=0, inplace=False, **kwargs):
     def head(self, n=5):
         return _col(self.to_dataframe().head(n))
 
+    # TODO: Categorical type isn't supported (due to PySpark's limitation) and
+    # some doctests related with timestamps were not added.
     def unique(self):
-        # Pandas wants a series/array-like object
-        return _col(self.to_dataframe().unique())
+        """
+        Return unique values of Series object.
+
+        Uniques are returned in order of appearance. Hash table-based unique,
+        therefore does NOT sort.
+
+        .. note:: This method returns newly creased Series whereas Pandas returns
+                  the unique values as a NumPy array.
+
+        Returns
+        -------
+        Returns the unique values as a Series.
+
+        See Examples section.
```

Comment:
nit: i don't think you need this since examples is literally one line below. i'm going to merge this and remove this line.